client/src/App.js

```diff
@@ -14,15 +15,25 @@ export default class App extends Component {
 
   addToSavedList = movie => {
     const savedList = this.state.savedList;
-    savedList.push(movie);
+    if (savedList.includes(movie)) {
+      return;
+    } else {
+      savedList.push(movie);
+    }
```

Comment:
Good conditional to prevent movies from being added to the list when they are already on there. Did you ever get the bug fixed up?